tests/lib/rules/sort-prop-types.js

```diff
@@ -486,18 +486,7 @@ ruleTester.run('sort-prop-types', rule, {
       line: 4,
       column: 5,
       type: 'Property'
-    }],
-    output: [
```

Comment:
similarly i wonder if we could comment out instead of remove all these `output`s - since they're all correct for when we re-enable the fixer.